spec/unit/middleware/rate_limiter_spec.rb

```diff
@@ -16,11 +16,12 @@ module Middleware
           }
         )
       end
-      let(:request_counter) { double }
+      let(:REQUEST_COUNTER) { double }
```

Comment:
This mocks the `request_counter` attribute in `BaseRateLimiter`, thus the previous name is correct.